security/pkg/pki/ca/controller/secret.go

```diff
@@ -50,6 +50,9 @@ const (
 	// The key to specify corresponding service account in the annotation of K8s secrets.
 	ServiceAccountNameAnnotationKey = "istio.io/service-account.name"
 
+	// The default SPIFFE URL value for identity domain
+	DefaultIdentityDomain = "cluster.local"
```

Comment:
The constant needs to be in some package other than `main` so it can be imported.
Moving it o main.go would mean duplicating the value (and ensuring it is consistent) somewhere in `security/pkg/pki/ca/controller/` since it is used in tests.
I can move it to another package under security/pkg, may be a `spiffe` identity package is needed  - it can also get some of the util files/functions.

Reply:
Right. We would like to put this and the spiffe formats into an "identity" package. We can do that in another PR. This PR is approved.